pkgs/os-specific/bsd/freebsd/pkgs/libedit.nix

```diff
@@ -0,0 +1,7 @@
+{ mkDerivation, libncurses-tinfo }:
+mkDerivation {
+  path = "lib/libedit";
```

Comment:
Is this different from all the libedit variants we have packaged at the top level?

Reply:
Yes. Various packages depend on filecomplete.h, which is not provided by the standard libedit. It _is_ provided by netbsd.libedit, but I would rather not mix freebsd and netbsd libs.